# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1468274895 25200
#      Mon Jul 11 15:08:15 2016 -0700
# Node ID d3cef63f79ac8fb8e97079dcc9867c002046efad
# Parent  0f6fc2ec3b1ab690c755b89a85608706a2cca835
* fminunc.m: Fix incorrect reference to fminunc as fminsearch (bug #48479)

diff --git a/scripts/optimization/fminunc.m b/scripts/optimization/fminunc.m
--- a/scripts/optimization/fminunc.m
+++ b/scripts/optimization/fminunc.m
@@ -79,17 +79,17 @@
 ## Optionally, @code{fminunc} can return a structure with convergence
 ## statistics (@var{output}), the output gradient (@var{grad}) at the
 ## solution @var{x}, and approximate Hessian (@var{hess}) at the solution
 ## @var{x}.
 ##
 ## Application Notes: If the objective function is a single nonlinear equation
 ## of one variable then using @code{fminbnd} is usually a better choice.
 ##
-## The algorithm used by @code{fminsearch} is a gradient search which depends
+## The algorithm used by @code{fminunc} is a gradient search which depends
 ## on the objective function being differentiable.  If the function has
 ## discontinuities it may be better to use a derivative-free algorithm such as
 ## @code{fminsearch}.
 ## @seealso{fminbnd, fminsearch, optimset}
 ## @end deftypefn
 
 ## PKG_ADD: ## Discard result to avoid polluting workspace with ans at startup.
 ## PKG_ADD: [~] = __all_opts__ ("fminunc");
